pkg/storage/replica_raft.go

```diff
@@ -1860,8 +1860,16 @@ func (r *Replica) processRaftCommand(
 		// succeeded. Note that we have taken precautions during command
 		// evaluation to avoid having mutations in the WriteBatch that affect
 		// the SSTable. Not doing so could result in order reversal (and missing
-		// values) here. If the key range we are ingesting into isn't empty,
-		// we're not using AddSSTable but a plain WriteBatch.
+		// values) here.
+		//
+		// Note that in the event of an ill-timed crash, the data may have been
+		// ingested (i.e. become "visible") without the Raft command having
+		// marked as applied. However, the node will only serve reads once a new
```

Comment:
Is this true with follower reads?

Reply:
Follower reads will see extra data, which is weird (and the data could be out-of-order with the raft log...) anyway, hard to figure what exactly a problem here would be, but closed timestamps would definitely go down the drain and everything is pretty undefined with sst ingestion anyway. I rearranged the comment and clarified.